### DISCUSSION TITLE
Some feedback

### DISCUSSION THREAD

**Discussion by Melonax** [bttv-android/bttv#508] [Ideas]
Hi! First off thank you for this app- it vastly improved my twitch experience over the past few months I've gone without a PC.

There's a few things I noticed and figured I would share in case you didn't know about them yet. None of these are too big of a deal but just so you know. So not really a feature request or bug report but some general feedback:

Emotes:
- Wide emotes show up tiny
- 0 width emotes don't stack 
- 7tv emotes arent animated when previewing/enlarging them, while bttv ones are

Chat:
- Would be nice to be able to see people's custom FFZ mod & vip badges, as well as an option to show/hide certain badges
- Would be nice to have the option for timestamps in chat, as well as adjusting further chat appearance (font size etc)
- Would love to see the pronouns plugin added somehow

General:
- It's a bit weird that bttv-android options show via both the chat settings and video settings buttons, but the gif emotes dropdown is only in one of those places. Also, you can use the phone's back button to get out of one of these menu's but not the other.
- There is no subscribe button (may have to do with in app purchases or something?)
- When auto redeem channel points is on, this takes priority over typing (keyboard disappears so I have to tap the chat window again to continue)

**Comment by FoseFx** [comment]
Hey! Thank you for taking the time to write this!

Some of these already have tracking issues:
- https://github.com/bttv-android/bttv/issues/411
- https://github.com/bttv-android/bttv/issues/199
- https://github.com/bttv-android/bttv/issues/497
- https://github.com/bttv-android/bttv/issues/444

I also created a couple new ones:
- https://github.com/bttv-android/bttv/issues/511
- https://github.com/bttv-android/bttv/issues/512
- https://github.com/bttv-android/bttv/issues/513

I don't think the font customization is in [Scope of the Project](https://github.com/bttv-android/bttv/blob/master/CONTRIBUTING.md#scope-of-the-project)

Unfortunately I do not have a lot of time right now so development is staling a bit.
